src/components/Icon/CheckIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
Non-blocking comment:
The `CheckIcon.tsx` look the same as the `TickIcon.tsx` except one has a circle around it. We could refactor either one of these components and set a prop to hide and show the circle and only use one component.

Reply:
Thanks, you're right and I didn't notice there was already one there. I can make a new ticket to combine, but will move this ticket on to QA for now